mix.exs

```diff
@@ -67,7 +67,7 @@ defmodule Lexical.MixProject do
           mix: :load
         ],
         include_executables_for: [:unix],
-        include_erts: false,
+        include_erts: true,
```

Comment:
this should go back to false.

Reply:
Are you sure?

> You may also set this option to false if you desire to use the ERTS version installed on the target. Note, however, that the ERTS version on the target must have the exact version as the ERTS version used when the release is assembled. Setting it to false also disables hot code upgrades. Therefore, :include_erts should be set to false with caution and only if you are assembling the release on the same server that runs it.

use `false`, we can't start the `start_lexical.sh` when version not matching.

Reply:
you're not understanding what the docs are saying. Including ERTS means that we include the entire runtime from the build environment, which means that this will not run on your computer if you download the build from githun, like vscode users do. We need to use the default runtime that users have on their computers. 
You also need to revert your changes to `start_lexical.sh`

Reply:
> Including ERTS means that we include the entire runtime from the build environment
I know this.

but the doc said: 

> that the ERTS version on the target must have the exact version as the ERTS version used when the release is assembled

Reply:
I think it means that `include_erts: false` requires the runtime version of erl to be consistent with the release version.

Reply:
I still thing this needs to be false. What we need to do is build a version for every major elixir / erlang release that we support, and the start_lexical script needs to be made smart about which environment it uses. 
So we need to build a matrix like:

Erlang:     Elixir
24            1.13, 1.14
25           1.13, 1.14
26           1.13, 1.14, (1.15)

Reply:
If this is true, the situation is even worse, we'll need to build releases for each elixir, erlang, O/S, *and* architecture we support.